web/packages/teleport/src/Integrations/Integrations.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useEffect } from 'react';
+import useAttempt from 'shared/hooks/useAttemptNext';
+import { Indicator, Box, Alert } from 'design';
+
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+// import useTeleport from 'teleport/useTeleport';
+import { integrationService } from 'teleport/services/integrations';
+
+// import { IntegrationsAddButton } from './IntegrationsAddButton';
+import { IntegrationList } from './IntegrationList';
+
+import type { Integration } from 'teleport/services/integrations';
+
+export function Integrations() {
+  const [items, setItems] = useState<Integration[]>([]);
+  const { attempt, run } = useAttempt('processing');
+
+  // const ctx = useTeleport();
+  // const canCreateIntegrations = ctx.storeUser.getIntegrationsAccess().create;
```

Comment:
Do we need these?

Reply:
yes b/c if the user doesn't have `create` access, then the `enroll pages` will not be renderable, so it's used as a flag to disable the button with hover tip `you do not have access etc`. we could just not render the button, but it doesn't align with how the other `add <some resource>` button works